<ui version="4.0" >
 <class>ExportSongDialog_UI</class>
 <widget class="QDialog" name="ExportSongDialog_UI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>198</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Export Whole Song to WAV</string>
  </property>
  <widget class="QWidget" name="layout2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>160</y>
     <width>445</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="spacing" >
     <number>0</number>
    </property>
    <property name="leftMargin" >
     <number>0</number>
    </property>
    <property name="topMargin" >
     <number>0</number>
    </property>
    <property name="rightMargin" >
     <number>0</number>
    </property>
    <property name="bottomMargin" >
     <number>0</number>
    </property>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType" >
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>126</width>
        <height>16</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="closeBtn" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>0</width>
        <height>26</height>
       </size>
      </property>
      <property name="text" >
       <string>&amp;Close</string>
      </property>
      <property name="shortcut" >
       <string>Alt+C</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType" >
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>126</width>
        <height>16</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>70</y>
     <width>441</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title" >
    <string/>
   </property>
   <widget class="QPushButton" name="okBtn" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>40</y>
      <width>72</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize" >
     <size>
      <width>0</width>
      <height>25</height>
     </size>
    </property>
    <property name="text" >
     <string>&amp;Export</string>
    </property>
    <property name="shortcut" >
     <string>Alt+E</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="m_pProgressBar" >
    <property name="geometry" >
     <rect>
      <x>90</x>
      <y>40</y>
      <width>341</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value" >
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="m_pSamplerateLbl" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>415</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text" >
     <string>textLabel1</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="TextLabel1" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>440</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>Export filename</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="exportNameTxt" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>40</y>
     <width>340</width>
     <height>24</height>
    </rect>
   </property>
   <property name="readOnly" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="browseBtn" >
   <property name="geometry" >
    <rect>
     <x>360</x>
     <y>40</y>
     <width>90</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>&amp;Browse...</string>
   </property>
   <property name="shortcut" >
    <string>Alt+B</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="0" margin="0" />
 <includes/>
 <resources/>
 <connections/>
</ui>
